database/migrations/2024_08_26_190935_create_encrypted_data.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('encrypted_data', function (Blueprint $table) {
+            $table->id();
+            $table->uuid('uuid')->unique()->nullable();
+            $table->string('field_uuid');
+            $table->unsignedInteger('request_id');
+            $table->string('iv');
+            $table->text('data');
+            $table->timestamps();
```

Comment:
Where the access permissions will be stored?
```
users_ids
groups_ids
```

Reply:
in the screen definition